multiversx/mx-chain-go
# Extend logs delegation contract

## Reasoning behind the pull request
- Extended logs generated by the `delegation` contract
  
## Proposed changes
- Extended the logs that are generated by `delegation` smart contract in case of `unDelegate` or `withdraw` 
- Upgraded the indexer version
  PR from indexer: https://github.com/multiversx/mx-chain-es-go/pull/196

## Testing procedure
- Create a `delegation` contract.
- Do `delegate` from another address.
- Do multiple `unDelegate` operations in multiple epochs from the same address.
- Check in the `delegators` index from the Elasticsearch database if there is a list of all the `unDelegate` operations for the test address.   

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
